features/apimgt/org.wso2.carbon.apimgt.rest.api.configurations.feature/pom.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
Incorrect year in the license header

Reply:
Fixed issue with https://github.com/RenukaFernando/carbon-apimgt/commit/b587c62656dbb084d324e343f09c7f9f8ab1672a